<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2022)/launch/sim_w1.launch" />
    <param name="desired_pos_x" value= "0.0" />
    <param name="desired_pos_y" value= "1.0" />
    <param name="frequency" type="double" value="1.0" />
    
    <node pkg="assignment_2_2022" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment_2_2022" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assignment_2_2022" type="bug_as.py" name="bug_action_service" output="screen" />
    <node pkg="assignment_2_2022" type="node_a_action_client.py" name="node_a_action_client" output="screen" launch-prefix="xterm -hold -e" />
    <node pkg="assignment_2_2022" type="node_b_print_goal_r_c.py" name="node_b_print_goal_r_c"  />
    <node pkg="assignment_2_2022" type="node_c_print_dist_spd.py" name="node_c_print_dist_spd" output="screen" launch-prefix="xterm -hold -e" />
</launch>
